src/exception.cr

```diff
@@ -49,8 +49,22 @@ class Exception
     end
   end
 
+  private def colorized_backtrace
+    if message.nil?
+      "\e[1m\e[31m" + self.class.name + "\e[0m"
+    else
+      "\e[1m\e[31m" + self.class.name + ": \e[0m\e[1m" + message.not_nil! + "\e[0m"
```

Comment:
Instead of building up strings with `+`, this method should probably take the io object as a parameter and use `<<`

Reply:
Fixed with new commit!